gpt4all-chat/chatapi.cpp

```diff
@@ -232,6 +234,8 @@ void ChatGPTWorker::handleReadyRead()
     if (!ok || code != 200) {
         m_chat->callResponse(-1, QString("\nERROR: 2 ChatGPT responded with error code \"%1-%2\" %3\n")
             .arg(code).arg(reply->errorString()).arg(reply->readAll()).toStdString());
+        m_chat->callResponse(-1, QString("\nERROR: 2 ChatAPI responded with error code \"%1-%2\" %3\n")
+                                     .arg(code).arg(reply->errorString()).arg(qPrintable(reply->readAll())).toStdString());
```

Comment:
It seems like this should have been a replacement instead of an addition?

Reply:
fixed